src/template-expr.adoc

```diff
@@ -199,59 +200,81 @@ https://github.com/amazon-ion/ion-docs/issues/201
 === Macro Invocation
 
 A macro definition can express its output in terms of other macros. Quite often, these will be
-macros provided by the Ion implementation,
-// —everything in System Macros 2023-05 is available by default—
-but they can also be acquired from other modules.
-
-// TODO link to system-macro chapter
-
-The S-expression syntax for macro invocation is similar to that of E-expressions.
-When a template is an S-expression and the first element is not the name of a special form, that
-element must instead be a _macro-ref_ and the template denotes a macro invocation.
-There are multiple sources of macros: the defining module’s internal environment (which is being
-incrementally extended with each definition), and the exported macros of modules loaded by the
-enclosing module or `$ion_encoding` directive.
-
-// TODO See Resolving Macro References: Encoding Modules 2023-05 for the relevant algorithm.
-
-The remaining elements of the S-expression are argument subforms that denote the inputs to the macro.
-These use normal Ion notation, but what’s syntactically acceptable is defined by the macro’s
-signature.
-
-The number of such subforms (that is, the invocation’s actual arity) must be equal to or greater
-than the macro’s minimum arity, and at most its maximum arity, when one exists.
-In other words, an invocation must contain one subform for each required parameter, followed by
-optional subforms for the remaining optional parameters.
+<<sec:sysmod,macros provided by the Ion implementation>>, but there are multiple sources of macros:
+
+* the defining module’s internal environment (which is being incrementally extended with each definition)
+* the macros exported from modules ``load``ed by the enclosing module
+* the macros exported from modules ``load``ed by the enclosing `$ion_encoding` directive
+
+The syntax for macro invocation in a template is similar to that of <<sec:eexprs,E-expressions>>.
```

Comment:
There is a lot in this section that is repeated. Could we explain macro invocations in a way that DRYs this?

Reply:
Perhaps, but there are also differences that make it hard to slot a unified description into either chapter. 
I'm inclined to leave that for later cleanup.